Infrastructure/Components/FilterByPrice.cs

```diff
@@ -0,0 +1,33 @@
+﻿using OpenQA.Selenium;
+using System.Linq;
+using Extensions;
+using OpenQA.Selenium.Interactions;
+
+namespace Infrastructure
+{
+    public class FilterByPrice : BaseComponent
+    {
+        private IWebElement PriceRange => ParentElement.WaitAndFindElement(By.CssSelector("#layered_price_range"));
+        private IWebElement RaisesTheMinPrice => ParentElement.FindElements(By.CssSelector("#layered_price_slider a")).FirstOrDefault();
+        private IWebElement ScrollPriceRange => ParentElement.WaitAndFindElement(By.CssSelector(".ui-slider-range.ui-widget-header.ui-corner-all"));
+
+        public FilterByPrice(IWebDriver driver, IWebElement parentElement) : base(driver, parentElement)
+        {
+        }
+
+        public string GetPriceRange() => PriceRange.Text;
+
+        public void ChangeFilterPrice(bool toRaiseTheMin)
+        {
+            int width = ScrollPriceRange.Size.Width;
+            Actions act = new Actions(Driver);
+            if (toRaiseTheMin)
+            {
+                act.MoveToElement(RaisesTheMinPrice).MoveByOffset((width / 3) - 10, 0).Click().Perform();
+            }else
+            {
+                act.MoveToElement(ScrollPriceRange).MoveByOffset((width / 3)+10, 0).Click().Perform();
+            }
+        }
+    }
```

Comment:
the way you handle the `ChangeFilterPrice` isnt good.

why does it get wether or raise the minimum as a bool, but doesn't get by how much?
you cant lower it back down.

same goes for the max.

it would be easier to create two separate methods, one for each element to move.
also the `...moveByOffset...` line isn't specifically for this component. you could extract it to your utilities with the `StandOn` function, that way you could click anything you want by offset.